docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/02 - Projects.md

```diff
@@ -0,0 +1,33 @@
+# Projects
+
+## Overview
+The **Project** represents a study in LORIS and can have different subprojects (see [Subproject setup](04 - Subprojects.md) page) which, in turn, can be affiliated with different visits (see [Timepoints setup](05 - Timepoints.md) page). **Projects** are not required in LORIS but it is recommended to enable this feature when possible.
```

Comment:
Aren't projects required now? Maybe I misunderstood but I thought it was not required but was made required in one of the last release? Just checking, I may be very wrong on this one ;).

Reply:
yep, I missed that